client/app/scripts/reducers/root.js

```diff
@@ -70,7 +71,7 @@ export const initialState = makeMap({
   updatePausedAt: null, // Date
   version: '...',
   versionUpdate: null,
-  websocketClosed: true,
+  websocketClosed: false,
```

Comment:
Nah, state variables should reflect the state without having to think about it. If we want to keep it flipped, I recommend changing the field name, e.g. `websocketHasBeenClosed` to make it explicit. Otherwise someone might flip it again in the future, thinking no websocket -> websocket closed.